.pylintrc

```diff
@@ -4,6 +4,7 @@
 #   public description of their style guide using 4)
 # - max-line-length changed to 120 because 80 was driving us crazy
 # - wrong-import-order re-enabled, because MT likes order among chaos
+# - disable cyclic-import check because it often has false positives
```

Comment:
This is the weirdest thing in this PR - newer pylint versions were giving cyclic-import warnings for ctakesclient, which to be fair, does have a weird import tree, where its __init__.py imports the submodules directly and then then import some submodules of their own. But I don't think it's cyclic? At any rate, it works - like python doesn't fail to import ctakesclient. So...  Just disabling this (I saw some other reports on their bug tracker of other people hitting similar things, not sure if we're just weirdos or pylint messed up.)